pkg/config/profile.go

```diff
@@ -11,33 +13,21 @@ type Profile struct {
 	Config *Config
 }
 
-// GetConfigField returns the configuration field for the specific profile
-func (p *Profile) GetConfigField(field string) string {
+// getConfigField returns the configuration field for the specific profile
+func (p *Profile) getConfigField(field string) string {
```

Comment:
@alexluong - how come this is lowercase when all other functions seem to begin with uppercase?

Reply:
That's how Golang handle public vs private methods, or internal package vs exported. It applies to functions, variables, structs, and everything you can imagine.

So if a function is lowercased, that means it can only be used in this package. So you can use `getConfigField` when you're in `pkg/config`, but when you're in another package you can't use this function.

I'm refactoring this a bit to limit the API surface of the config module.